cli.ts

```diff
@@ -1,19 +1,18 @@
 #!/usr/bin/env node
 
-const fse = require("fs-extra");
+import * as fse from "fs-extra";
```

Comment:
This is not an esm module, `import`s don't work. And let's stick with `require` to keep a uniform import syntax

Reply:
yep, sounds good. I will do this as first thing tomorrow(it's night here).

Reply:
Alright 👍